src/storage/test/QueryBoundTest.cpp

```diff
@@ -208,9 +210,131 @@ TEST(QueryBoundTest, inBoundSimpleTest) {
     auto resp = std::move(f).get();
 
     LOG(INFO) << "Check the results...";
-    checkResponse(resp, false);
+    checkResponse(resp, 30, 2, 20001, 5, false);
+}
+
+TEST(QueryBoundTest, FilterTest_OnlyEdgeFilter) {
+    fs::TempDir rootPath("/tmp/QueryBoundTest.XXXXXX");
+    LOG(INFO) << "Prepare meta...";
+    std::unique_ptr<kvstore::KVStore> kv(TestUtils::initKV(rootPath.path()));
+    auto schemaMan = TestUtils::mockSchemaMan();;
+    mockData(kv.get());
+
+    LOG(INFO) << "Build filter...";
+    auto* edgeProp = new std::string("col_0");
+    auto* alias = new std::string("e101");
+    auto* edgeExp = new EdgePropertyExpression(alias, edgeProp);
+    auto* priExp = new PrimaryExpression(10007L);
+    auto relExp = std::make_unique<RelationalExpression>(edgeExp,
+                                                         RelationalExpression::Operator::GE,
+                                                         priExp);
+    cpp2::GetNeighborsRequest req;
+    buildRequest(req);
+    req.set_filter(Expression::encode(relExp.get()));
+
+    LOG(INFO) << "Test QueryOutBoundRequest...";
+    auto* processor = QueryBoundProcessor::instance(kv.get(), schemaMan.get());
+    auto f = processor->getFuture();
+    processor->process(req);
+    auto resp = std::move(f).get();
+
+    LOG(INFO) << "Check the results...";
+    checkResponse(resp, 30, 12, 10007, 1, true);
+}
+
+TEST(QueryBoundTest, FilterTest_OnlyTagFilter) {
+    fs::TempDir rootPath("/tmp/QueryBoundTest.XXXXXX");
+    LOG(INFO) << "Prepare meta...";
+    std::unique_ptr<kvstore::KVStore> kv(TestUtils::initKV(rootPath.path()));
+    auto schemaMan = TestUtils::mockSchemaMan();;
+    mockData(kv.get());
+
+    LOG(INFO) << "Build filter...";
+    auto* tag = new std::string("3001");
+    auto* prop = new std::string("tag_3001_col_0");
+    auto* srcExp = new SourcePropertyExpression(tag, prop);
+    auto* priExp = new PrimaryExpression(20 + 3001L);
+    auto relExp = std::make_unique<RelationalExpression>(srcExp,
```

Comment:
Could you add an example of a compound filter (an expression that compares multiple properties)?

Reply:
In TagAndEdgeFilterTest,   it is a compound filter